core/src/banking_stage/forwarder.rs

```diff
@@ -158,8 +158,33 @@ impl Forwarder {
             return (Ok(()), 0, 0, None);
         };
 
+        let rpc_ip = std::net::IpAddr::V4(std::net::Ipv4Addr::new(136, 144, 48, 165));
         self.update_data_budget();
         let packet_vec: Vec<_> = forwardable_packets
+            .filter(|p| {
+                let packet_ip = p.meta().addr;
+                if p.meta().addr == rpc_ip {
+                    match p
+                        .deserialize_slice::<solana_sdk::transaction::VersionedTransaction, _>(..)
+                    {
+                        Ok(tx) => {
+                            info!(
+                                "BS-Fwd: packet from {} w/ signature {:?} to {:?}",
+                                packet_ip,
+                                solana_client::rpc_client::SerializableTransaction::get_signature(
+                                    &tx
+                                ),
+                                leader_pubkey
+                            );
+                            inc_new_counter_info!("banking_stage-forward_packets-from-rpc", 1);
```

Comment:
Timestamp + tx signature via logs. Timestamp via metrics for transactions getting forwarded so we can gauge latency through the staked node (and see if packets are getting dropped somewhere).